app/assets/scripts/views/location.js

```diff
@@ -239,6 +242,83 @@ var Location = React.createClass({
     );
   },
 
+  renderMetadata: function () {
+    const { loc, loc: { data: { metadata } } } = this.props;
+
+    const exclude = [
+      'id',
+      'coordinates',
+      'city',
+      'country',
+      'instruments',
+      'parameters',
+      'attribution'
+    ];
+
+    const allProperties = Object.keys(locationSchema.properties)
+      .filter((key) => {
+        return !exclude.includes(key) && metadata[key];
+      });
+
+    const propertiesMain = [];
+    const propertiesSec = [];
+    const length = Math.ceil(allProperties.length / 2);
+
+    allProperties.forEach((key, i) => {
+      const prop = locationSchema.properties[key];
+      prop.key = key;
+      let val = metadata[prop.key];
+
+      if (prop.format && prop.format === 'date-time') {
+        val = moment.utc(val).format('YYYY/MM/DD');
+      }
+      if (prop.type && prop.type === 'boolean') {
+        val = val ? 'Yes' : 'No';
+      }
+
+      const sectionIndex = Math.floor(i / length);
+
+      switch (sectionIndex) {
+        case 0: {
+          propertiesMain.push(<dt key={`${key}-${prop.title}`}>{`${prop.title}`}</dt>);
```

Comment:
Unnecessary template literal around `prop.title`
```suggestion
          propertiesMain.push(<dt key={`${key}-${prop.title}`}>{prop.title}</dt>);
```